bot.js

```diff
@@ -100,4 +97,15 @@ function reply(message) {
 	  }
 	  conversation.innerHTML += "<p><strong>You:</strong> " + message + "</p>";
 	  conversation.innerHTML += "<p><strong>Eliza:</strong> " + answer + "</p>";
-}
\ No newline at end of file
+}
+
+function applyReplacements(substitution) {
+	for (var j = 0; j < replacements.length; j++) 
```

Comment:
I think you have a 'tab' character here, but a repeated 'space' everywhere else. This can cause issues if people have different setting for the lenght of tab in their editors.